app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] When running your project locally I am getting the following error please fix it 👇 


https://user-images.githubusercontent.com/75126481/225759055-ef624eca-ad1b-4c26-b568-cd3560a0ec22.mov